rsconnect/metadata.py

```diff
@@ -475,6 +471,10 @@ def resolve(self, server, app_id, app_mode):
         app_store_version = metadata.get("app_store_version")
         return app_id, app_mode, app_store_version
 
+    @property
+    def current_app_store_version(self):
+        return 1
+
```

Comment:
We should put current_app_store_version - let's rename it 'version' for brevity - as a default parameter with default of 1 in __init__, then save it in self.version. We probably don't need this property at this point since we can just init the class (e.g. app_store = AppStore() ) and check app_store.version directly.